core/cautils/fileutils.go

```diff
@@ -97,22 +102,22 @@ func LoadResourcesFromKustomizeDirectory(ctx context.Context, basePath string) (
 	return sourceToWorkloads, kustomizeDirectoryName
 }
 
-func LoadResourcesFromFiles(ctx context.Context, input, rootPath string) map[string][]workloadinterface.IMetadata {
+func LoadResourcesFromFiles(ctx context.Context, input, rootPath string) (map[string][]workloadinterface.IMetadata, error) {
 	files, errs := listFiles(input)
 	if len(errs) > 0 {
 		logger.L().Ctx(ctx).Warning(fmt.Sprintf("%v", errs))
 	}
 	if len(files) == 0 {
 		logger.L().Ctx(ctx).Error("no files found to scan", helpers.String("input", input))
-		return nil
+		return nil, errors.New(fmt.Sprintf(ErrNoFilesToScan, input))
```

Comment:
pls mind the linter comments

Reply:
I don't see any linter issue.

Reply:
![image](https://github.com/kubescape/kubescape/assets/84019060/40fb9fca-608b-4c15-b220-bf70a334c6c6)
can you be more spevcific?

Reply:
![image](https://github.com/kubescape/kubescape/assets/84905812/c775948f-1270-447b-b4b1-2262badfa8d2)
You can see them on your commit, for example: https://github.com/kubescape/kubescape/pull/1296/commits/b730ef51543e9f7d84db6137870f1454c38a104e